src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeOptionsFormDataProvider.php

```diff
@@ -52,16 +52,14 @@ public function __construct(DataConfigurationInterface $employeeOptionsConfigura
      */
     public function getData()
     {
-        return [
-            'options' => $this->employeeOptionsConfiguration->getConfiguration(),
-        ];
+        return $this->employeeOptionsConfiguration->getConfiguration();
```

Comment:
BC, rigth ?

Reply:
Nope, it's a bug, not a bc breaks as the behavior is not the right one.

Reply:
Removing **options** key in return, will break where it is expected right?

Reply:
> Removing **options** key in return, will break where it is expected right?

No, it's a missing change from an old PR, it's currently just a bug and shouldn't returns the options parameter ^^